apps/sms/views/shared/test/unit/navigation_test.js

```diff
@@ -787,9 +787,11 @@ suite('navigation >', function() {
       function(done) {
         Object.defineProperty(document, 'readyState', { value: 'interactive' });
 
-        // For the split-view case Navigation.init waits for the setReady call
-        // that is made in the suite setup function above.
-        Navigation.init().then(done, done);
+        var NonReadyNavigation = NavigationFactory(fakeWindow);
```

Comment:
Because ```setReady``` is called in setup so that other tests within this suite work as expected. I had two choices to make it fail on master, either this, or move it to another suite - chose the simplest - what do you prefer?